OWML.Launcher/App.cs

```diff
@@ -42,7 +42,10 @@ public void Run(string[] args)
 
             CopyGameFiles();
 
-            ListenForOutput();
+            if (!CommandLineArguments.HasArgument("consolePort"))
+            {
+                ListenForOutput();
```

Comment:
Since the OWML console window is now empty, we could hide it. But I don't know how 😫

Reply:
Well, we could just quit :P
`Environment.Exit(0);`

Reply:
So I'd call Exit right after `StartGame(args)`?

Reply:
I guess :)